data/src/main/java/org/oppia/data/backends/gae/ModifyJsonInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.*
+
+import java.io.IOException
+
+class ModifyJsonInterceptor : Interceptor {
+
+  @Throws(IOException::class)
+  override fun intercept(chain: Interceptor.Chain): Response {
+    val request = chain.request()
+    val response = chain.proceed(request)
+
+    if (response.code() == 200) {
```

Comment:
Please move ``200`` to a module constant to provide context to it so that it's not a seemingly magic number. If Retrofit has a constant for it, that would be even better.

Reply:
Created new Constants file at module level and introduced 200 as a constant.